yarn.lock

```diff
@@ -361,6 +361,13 @@
   dependencies:
     tslib "^2.1.0"
 
+"@capacitor/core@^3.4.1":
```

Comment:
if we remove your local dependency, i guess there is no need to update the yarn lock in this PR as no new dependencies are added?